### PR TITLE
Recreate Deepin related ruleset.

### PR DESCRIPTION
**Pull request by lqf96**
Including "*.deepin.com", "*.deepin.org" and "*.linuxdeepin.com" domains.

### CODE REVIEW

File: src/chrome/content/rules/Deepin.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <!-- Redirects to *.linuxdeepin.com -->
+    <rule from="^http://(packages|cdimage)\.deepin\.com/" to="https://$1.linuxdeepin.com/" />
+    <rule from="^http://(packages)\.deepin\.org/" to="https://$1.linuxdeepin.com/" />
+    <!-- Redirects directly to HTTPS -->
+    <rule from="^http:" to="https:" />
+    
+    <!--
+    Mismatch:
+    wiki.deepin.org
+    blog.deepin.org
```
Comment (J0WI) [inline src/chrome/content/rules/Deepin.xml]: Seems to working now (https://github.com/EFForg/https-everywhere/pull/4230/files#diff-b460b3e9ba52aa834775ba7d1b087c00R11)

**Comment by J0WI** [comment]
I just realized that there is also a img3.linuxdeepin.com.
Please also redirect deepin.org/com to www.deepin.org/com. This is because the server side redirect doesn't support https.

**Comment by lqf96** [comment]
OK, I've updated all of that.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**